future_challenges_for_RSEng.tex

```diff
@@ -516,9 +516,6 @@ \subsection{Ethics and Social Considerations}
 research software developed by RSEs should adhere to the FAIR4RS principles~\cite{FAIR4RS}.
 Members of the SE community have also called for their field to embrace Open Science best practices\cite{OliveiraJr2024,Druskat2025}.
 
-A code of ethics and professional conduct for RSEs should be developed by institutions together with RSEs themselves.
-This can then form the basis for institutional policies and guidelines as well as serve as a background for individual developers to consider how software can be used.
```

Comment:
Do we now completely drop the "code of ethics and professional conduct for RSEs", or is this represented somewhere else?

Reply:
very true, indeed:

> RSEs need to be aware of these issues and develop their own ``red lines''
which need to be supported by organisational and professional guidelines.